.github/workflows/publish.yaml

```
@@ -0,0 +1,13 @@
+name: Publish Image
+on:
+  release:
+    types:
+    - released
```

Given our CI design, publish happens on merge into `master`. _IF_ you feel you need to be cautious and have a manual release, then you would do that with a `release.yaml` that promotes the image. I don't think we need to be that cautious for this project, though.

```suggestion
on:
  push:
    branches:
    - master
```